changelog.d/sca-block-bug.fix

```diff
@@ -1 +0,0 @@
-Fixed bug where unreachable supply chain findings could be marked blocking
```

Comment:
@mmcqd I've deleted this file. Not sure why it was still there still a month after (probably
because it didn't respect the towncrier format for files in this dir.